kotlin/http4k/gradle.properties

```diff
@@ -0,0 +1,4 @@
+kotlin_version=1.4.10
```

Comment:
could we `1.4.+` in order to have the last patched version in **1.4** ?

Reply:
Sure can!